### PR TITLE
Remove no need $ character

### PR DESCRIPTION
**Pull request by ryuuc**
In this PR I removed some no need $ characters since there are plain strings but no interpolated strings.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=34459) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=34459) before we can accept your contribution.<br/><br/>:x: ryuuc [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=34459)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=34459) it.</sub>

File: src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ClassifiedSpan/ClassifiedSpanVerifier.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (Youssef1313) [inline src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ClassifiedSpan/ClassifiedSpanVerifier.cs:1]: What's the change here? Does this remove the "BOM character"? I think this should be reverted.

File: src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/TagHelperSpan/TagHelperSpanVerifier.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (Youssef1313) [inline src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/TagHelperSpan/TagHelperSpanVerifier.cs:1]: Same comment here

File: src/Http/Routing/test/UnitTests/DefaultParameterPolicyFactoryTest.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (Youssef1313) [inline src/Http/Routing/test/UnitTests/DefaultParameterPolicyFactoryTest.cs:1]: Same comment here

File: src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ClassifiedSpan/ClassifiedSpanVerifier.cs (hunk 1 above)
Comment (pranavkm) [reply src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ClassifiedSpan/ClassifiedSpanVerifier.cs:1]: Outside of files that are part of the project templates. I don't particularly mind these sort of changes. Encoding of files isn't consistent across the repo and some of these changes often update the file encoding to match what's in .editorconfig

**Review by pranavkm — APPROVED** [review]
Did you use an analyzer to discover these?

**Comment by pranavkm** [comment]
Thanks for the PR!

**Comment by ryuuc** [comment]
> Did you use an analyzer to discover these?

No, I just noticed it when I explore some health check code.

**Pull request merged**